src/sandbox/policies/greedy_policy.py

```diff
@@ -0,0 +1,22 @@
+
+
+from typing import Iterable
+from sandbox.algorithms.algorithm import Action
+from sandbox.enviroments.base.observation import Observation
+from sandbox.enviroments.multi_armed_bandit.policy import DeterministicPolicy
+from sandbox.value_estimates.value_estimates import ActionValueEstimates, StateValueEstimates
+
+class GreedyPolicy(DeterministicPolicy):
+
+    def _policy_for_action_value_estimate(self, observation: Observation, actions: Iterable[Action], estimates: ActionValueEstimates):
+        return max([(action, estimates[(observation, action)])
+            for action in actions
+        ], key=lambda i: i[1])[0]
+
+    def _policy_for_value_estimate(self, next_observations: Iterable[Observation], estimates: StateValueEstimates):
```

Comment:
According to the book, policy is a
> decision-making rule

![image](https://user-images.githubusercontent.com/21079319/165658800-62f5081f-9841-4c77-b97f-fc9daa6999e6.png)


So I represent it as a function. 

As far as i understand, there are basically 2 scenarios, under which decision could be made: one where we have some `ActionValueEstimates` and one where we have `StateValueEstimates`, so for every scenario I added a specific function.